persistence/album_repository.go

```diff
@@ -158,6 +163,18 @@ func (r *albumRepository) Refresh(ids ...string) error {
 	return err
 }
 
+func getMinYear(years string) int {
```

Comment:
Something here is a bit weird, either:
- we're sure that all the dates are correctly formatted, and they should be stored in a proper format instead of strings
- we're unsure that the dates are correctly formatted, and the error case of `strconv.Atoi` should be properly handled

Reply:
It is the former: this function is called with the results of `group_concat(f.year, ' ')` (line 120). It will always receive a well-formated list of years as string. 

I could have embedded the function in the `Refresh` method, but I rather separate this kind of small functions so they can be thoroughly tested. And the [tests](https://github.com/deluan/navidrome/blob/69dc4d97b3a6e510509b6db1305a43bea86fa309/persistence/album_repository_test.go#L76) serve as a "contract" for the function usage

Reply:
And if the `years` have non-number strings, `strconv.Atoi` will return 0 (and an err), which is fine for our purposes. See the first test case:
https://github.com/deluan/navidrome/blob/69dc4d97b3a6e510509b6db1305a43bea86fa309/persistence/album_repository_test.go#L77